Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py

```diff
@@ -16,25 +16,25 @@
 # this calculation taken from the EOS equations in [1]
 def calculate_pressure(d_spacing, temp):
     v00 = 121.41813  # zero pressure unit cell volume at (0) (A^3)
```

Comment:
I wonder if we could extract these constants as module level variables with capitalized names to indicate they are constants that might given them more meaning without the comments?

Reply:
Tom's off now until next Thurs and Chris Ridley has asked that we put this fix through quickly. Could Tom do the change to these variables as a separate issue to enable this fix to go into the nightly soon?

Reply:
Yeah of course. Merging now.